webapi/src/test/scala/org/knora/webapi/e2e/v2/SearchRouteV2R2RSpec.scala

```diff
@@ -2499,68 +2499,6 @@ class SearchRouteV2R2RSpec extends R2RSpec {
             }
         }
 
-        "do a Gravsearch query for a letter that links to a person with a specified name (optional)" in {
```

Comment:
why have you removed these three tests?

Reply:
As explained in the description of this PR:

> Remove some tests from `SearchRouteV2R2RSpec` using `OPTIONAL`, because they make it difficult to add new test data, and seem redundant anyway (the `OPTIONAL` doesn't affect the results).

Reply:
The problem is that these tests are so vague that when you add new test data, the existing vague tests find the new data and this makes them fail.

Reply:
And because the `OPTIONAL` has no effect on the results, those tests are the same as other existing tests without the `OPTIONAL`.